apps/onboarding-cdc/src/main/java/it/pagopa/selfcare/onboarding/event/mapper/OnboardingMapper.java

```diff
@@ -0,0 +1,28 @@
+package it.pagopa.selfcare.onboarding.event.mapper;
+
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+import org.openapi.quarkus.onboarding_functions_json.model.Onboarding;
+
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.UUID;
+
+@Mapper(componentModel = "cdi", imports = UUID.class)
+public interface OnboardingMapper {
+    @Mapping(target = "createdAt", source = "createdAt", qualifiedByName = "toOffsetDateTime")
+    @Mapping(target = "activatedAt", source = "activatedAt", qualifiedByName = "toOffsetDateTime")
+    @Mapping(target = "expiringDate", source = "expiringDate", qualifiedByName = "toOffsetDateTime")
+    @Mapping(target = "updatedAt", source = "updatedAt", qualifiedByName = "toOffsetDateTime")
+    Onboarding toEntity(it.pagopa.selfcare.onboarding.event.entity.Onboarding model);
+    @Named("toOffsetDateTime")
+    default OffsetDateTime map(String value) {
+        LocalDateTime localDateTime = LocalDateTime.parse(value);
+        ZonedDateTime zonedDateTime = localDateTime.atZone(ZoneId.of("Europe/London"));
```

Comment:
why London?

Reply:
I saw that on the DB we save the dates as UTC +0 and searching on the internet it should be London, I don't actually know if my reasoning is correct

Reply:
Changed to default